application/helpers/common_helper.php

```diff
@@ -5041,3 +5041,27 @@ function standardDeviation(array $numbers): float
 
     return sqrt($variance / $numberOfElements);
 }
+
+/**
+ * Returns a random number
+ * @param int $max The number maximun of string
+ * @param int $min The number minimum of string
+ * @return int
+ */
+function safeRandom($max, $min = 0)
```

Comment:
This 2 function are only used for this plugin, right ?

Adding it as private static function inside plugin seems better in my opinion.

Reply:
I don't care that much about private vs common but why do we need safe random at all? Can't we just use mt_rand? Or PHP version issue?